introduction.md

```diff
@@ -1,2 +1,3 @@
 "Hello! I am Martin and I really want to learn something new."
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?